gql_code_builder/lib/src/operation/data.dart

```diff
@@ -3,48 +3,85 @@ import "package:built_collection/built_collection.dart";
 import "package:code_builder/code_builder.dart";
 import "package:gql/ast.dart";
 import "package:gql_code_builder/src/common.dart";
+import "package:gql_code_builder/source.dart";
+
+class SourceSelections {
```

Comment:
I needed to keep track of the source URL along with the selections. If you feel like there's a better way to do this, please let me know.

Reply:
I think this makes sense. Please make it private.